### PR TITLE
Add firmware.envelope to host systems.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: tools/firmware.toit [hunk 1]
```diff
@@ -122,10 +125,10 @@ write-file-or-print --path/string? output/string -> none:
     print output
 
 main arguments/List:
-  firmware-cmd := build-command
+  firmware-cmd := build-command --create-esp32-only
```
Comment (kasperl) [inline tools/firmware.toit:128]: So this way 'tools/firmware' keeps it old interface, but 'toit firmware' is updated?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tools/firmware.toit (hunk 1 above)
Comment (floitsch) [reply tools/firmware.toit:128]: correct.

**Review by kasperl — APPROVED** [review]
Still looks good!

**Pull request merged**